modules/custom/apigee_kickstart_m10n_add_credit/apigee_kickstart_m10n_add_credit.info.yml

```diff
@@ -1,5 +1,5 @@
-name: Apigee Kickstart Monetization
-description: "Handles monetization for Apigee Kickstart. Adds and configures a default product type for adding credit."
+name: Apigee Kickstart Monetization (Deprecated)
```

Comment:
In D7, you could hide modules, is that something we should do here?

Reply:
Yep, I've added `hidden: true` to it's yml file.